src/Keboola/StorageApi/GCSUploader.php

```diff
@@ -58,6 +58,7 @@ public function getLastReceivedToken()
         $this->gcsClient = new GoogleStorageClient([
             'projectId' => $options['projectId'],
             'credentialsFetcher' => $this->fetchAuthToken,
+            'requestTimeout' => 120,
```

Comment:
120 ma nastaveny aj driver https://github.com/keboola/php-storage-driver-bigquery/blob/358a59d25bf158ca4ea6a54dc9c45a1f67e0c764/src/GCPClientManager.php#L156

Reply:
Tady je to imho dost málo, na S3 máme 500 https://github.com/keboola/storage-api-php-client/blob/master/src/Keboola/StorageApi/Client.php#L1906C30-L1906C33 takže bych dal taky 500